packages/ui/src/components/GalleryInput/index.native.tsx

```diff
@@ -0,0 +1,531 @@
+import {
```

Comment:
Should extract everything duplicated from `MessageInput` in some sensible way.

Reply:
Combined the inputs in latest commits